<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиентская панель</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="profileTab">
       <attribute name="title">
        <string>Профиль</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name=" nameLabel">
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="nameValueLabel"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="emailValueLabel"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="myMembershipsTab">
       <attribute name="title">
        <string>Мои абонементы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="membershipTable">
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начало</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Конец</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="availableMembershipsTab">
       <attribute name="title">
        <string>Доступные абонементы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="availableMembershipsTable">
          <column>
           <property name="text">
            <string>Тип</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начало</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Конец</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buyMembershipButton">
            <property name="text">
             <string>Купить абонемент</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="trainerServicesTab">
       <attribute name="title">
        <string>Тренерские услуги</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_trainerServices">
        <item>
         <widget class="QTableWidget" name="trainerServicesTable">
          <column>
           <property name="text">
            <string>Вид услуги</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Срок</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Начало</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Конец</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_trainerServices">
          <item>
           <spacer name="horizontalSpacer_trainerServices">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buyTrainerServiceButton">
            <property name="text">
             <string>Купить услугу</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="visitsTab">
       <attribute name="title">
        <string>Посещения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="visitsTable">
          <column>
           <property name="text">
            <string>Дата и время</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_visits">
          <item>
           <spacer name="horizontalSpacer_visits">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="markVisitButton">
            <property name="text">
             <string>Отметить посещение</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="1"/>
 <resources/>
 <connections/>
</ui>